unique_toolkit/unique_toolkit/framework_utilities/langchain/client.py

```diff
@@ -23,23 +22,24 @@ def __init__(self):
     raise LangchainNotInstalledError()
 
 
-def get_client(env_file: Path | None = None) -> ChatOpenAI:
+def get_client(
+    unique_settings: UniqueSettings, model: str = "AZURE_GPT_4o_2024_0806"
+) -> ChatOpenAI:
```

Comment:
This is a breaking change. Hopefully no one uses it. Perhaps best to mention it in change log just to be sure.